### DISCUSSION TITLE
Problem: Arrows don't do enough damage.

### DISCUSSION THREAD

**Discussion by ZayyRMC** [FunkinCrew/Funkin#1637] [Ideas]
Arrows don't do enough damage and allow spamming. It makes the levels too easy. (I preferably play in hard mode all the time). It doesn't have to do too much damage. Just not too little where you can spam all 4 arrows and still have over 75% of your health during a "hard" part of the song.

**Comment by aflacc** [comment]
I feel like what should happen is maybe you lose more health on higher difficulties. Like maybe easy would be a little lesss than what we have now, normal would be what we currently have and hard would be more that what we have now. I might do this in ProjectFNF, my engine as well

**Comment by Windows1Door2Daniel** [comment]
People who play on hard think the game should be more hard

What about people who play in EASY and NORMAL?

**Comment by aflacc** [comment]
well make the health drain less depending on the difficulty
